<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Turtle::LibraryBrowser</class>
 <widget class="QWidget" name="Turtle::LibraryBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Browser</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="documentView">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="pictureLabel">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="descriptionField">
       <property name="font">
        <font>
         <family>Courier</family>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
